src/query/v2/common.hpp

```diff
@@ -103,34 +103,40 @@ concept RecordAccessor =
 inline void HandleSchemaViolation(const storage::v3::SchemaViolation &schema_violation, const DbAccessor &dba) {
   switch (schema_violation.status) {
     case storage::v3::SchemaViolation::ValidationStatus::VERTEX_HAS_NO_PRIMARY_PROPERTY: {
+      // TODO(antaljanosbenjamin)
```

Comment:
Do we call this function at any time currently?

Reply:
Nope will remove it, along with other `PropsSetChecked`, since it is unused too